docs/getting-started-guides/docker-multinode/master.sh

```diff
@@ -39,6 +38,12 @@ if [ "$(id -u)" != "0" ]; then
     exit 1
 fi
 
+# Make default eth0
+if [ -z ${FLANNEL_IF} ]; then
+    echo "Using eth0 as default"
+    FLANNEL_IF="eth0"
```

Comment:
Ok, i have rewrite the all script on my side to make it clearer, so i'll update the PR